charts/datadog/templates/_orchestrator_explorer_config.yaml

```diff
@@ -0,0 +1,42 @@
+{{- define "orchestratorExplorer-add-default-config" -}}
+
+{{- $useDefaultConfig := true -}}
+
+{{/*
+If custom config is provided, then we don't use default config.
+*/}}
+{{- range $k, $v := .Values.clusterAgent.confd -}}
+{{- if regexMatch "^orchestrator.yaml.*"  $k  -}}
+{{- $useDefaultConfig = false -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+If customResourcesCollection is empty, then we don't use default config.
+*/}}
+{{- if eq $useDefaultConfig true  -}}
+{{- if eq (len $.Values.datadog.orchestratorExplorer.customResourcesCollection) 0 }}
+{{- $useDefaultConfig = false -}}
+{{- end -}}
+{{- end -}}
+
+{{- $useDefaultConfig -}}
+
+{{- end -}}
+
+
+{{- define "orchestratorExplorer-config" -}}
+
+{{- if eq (include "orchestratorExplorer-add-default-config" .) "true" -}}
+orchestrator.yaml.default: |-
```

Comment:
i wonder if it's a bit misleading to call it `default` since it's configured by the user input?

Reply:
It sounds reasonable, I renamed it 
https://github.com/DataDog/helm-charts/blob/64823a361eff2d4c4f5536b2d887ec081df42ee4/charts/datadog/templates/_orchestrator_explorer_config.yaml#L40